weaviate/collections/batch/base.py

```diff
@@ -231,74 +239,22 @@ def periodic_check() -> None:
                 self.__shut_background_thread_down is not None
                 and not self.__shut_background_thread_down.is_set()
             ):
-                if not self.__dynamic_batching:
+                if isinstance(self.__batching_mode, _FixedSizeBatching):
                     refresh_time: float = 0.1
+                elif isinstance(self.__batching_mode, _RateLimitedBatching):
+                    if (
+                        time.time() - self.__last_request
+                        < self.__fix_rate_batching_base_time // self.__concurrent_requests
```

Comment:
Since `self.__fix_rate_batching_base_time` has units of `s/request` shouldn't this be `*` instead of `//`?

Reply:
I added some explanation to where these variables are initialized. Could you check if that makes it clear? :)

Happy to add more if needed